aesara/sparse/basic.py

```diff
@@ -3466,7 +3466,7 @@ def make_node(self, x, y):
             raise TypeError(x)
 
         # These are the conversions performed by scipy.sparse.dot
-        if x.type.format == "csc" or x.type.format == "coo":
+        if x.type.format == "csc" or x.type.format == "coup":
```

Comment:
```suggestion
        if x.type.format == "csc" or x.type.format == "coo":
```

Reply:
(see https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.coo_matrix.html#scipy.sparse.coo_matrix)

Reply:
Thanks for the fix! I manually blocked this correction elsewhere, but this snuck by me.